Formula/gauge.rb

```diff
@@ -42,6 +42,12 @@ def install
     end
   end
 
+  def caveats; <<-EOS.undent
```

Comment:
@ilovezfs, thanks for the review. I was working with @Apoorva-GA on this one. 

What's the best way to give the user a message such as this? This won't be very verbose, we understand that `brew install` should not clutter the stdout.

I took a hint from [another formula](https://github.com/Homebrew/homebrew-core/blob/master/Formula/mosquitto.rb#L31)